Using MAPPING_STR=11122200, VERSION_DIR=v4_engoff
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.sh: line 6: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.sh: line 8: sudo: command not found
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q12_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q12_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q12_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q12_t0.sh: line 6: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q13_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Get:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease [114 kB]
0% [2 InRelease 5583 B/114 kB 5%]                                 0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]100% [Working]              Fetched 114 kB in 1s (132 kB/s)
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q14_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q14_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q14_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q15_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
0% [Waiting for headers]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3 set to manually installed.
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: numpy>=1.20.3; python_version < "3.10" in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.sh: line 6: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.sh: line 9: sudo: command not found
Collecting pip
  Downloading pip-23.3.1-py3-none-any.whl (2.1 MB)
[?25l[K     |▏                               | 10 kB 11.6 MB/s eta 0:00:01[K     |▎                               | 20 kB 12.7 MB/s eta 0:00:01[K     |▌                               | 30 kB 18.4 MB/s eta 0:00:01[K     |▋                               | 40 kB 4.9 MB/s eta 0:00:01[K     |▊                               | 51 kB 4.5 MB/s eta 0:00:01[K     |█                               | 61 kB 5.3 MB/s eta 0:00:01[K     |█                               | 71 kB 6.2 MB/s eta 0:00:01[K     |█▎                              | 81 kB 6.3 MB/s eta 0:00:01[K     |█▍                              | 92 kB 7.1 MB/s eta 0:00:01[K     |█▌                              | 102 kB 7.0 MB/s eta 0:00:01[K     |█▊                              | 112 kB 7.0 MB/s eta 0:00:01[K     |█▉                              | 122 kB 7.0 MB/s eta 0:00:01[K     |██                              | 133 kB 7.0 MB/s eta 0:00:01[K     |██▏                             | 143 kB 7.0 MB/s eta 0:00:01[K     |██▎                             | 153 kB 7.0 MB/s eta 0:00:01[K     |██▌                             | 163 kB 7.0 MB/s eta 0:00:01[K     |██▋                             | 174 kB 7.0 MB/s eta 0:00:01[K     |██▉                             | 184 kB 7.0 MB/s eta 0:00:01[K     |███                             | 194 kB 7.0 MB/s eta 0:00:01[K     |███                             | 204 kB 7.0 MB/s eta 0:00:01[K     |███▎                            | 215 kB 7.0 MB/s eta 0:00:01[K     |███▍                            | 225 kB 7.0 MB/s eta 0:00:01[K     |███▋                            | 235 kB 7.0 MB/s eta 0:00:01[K     |███▊                            | 245 kB 7.0 MB/s eta 0:00:01[K     |███▉                            | 256 kB 7.0 MB/s eta 0:00:01[K     |████                            | 266 kB 7.0 MB/s eta 0:00:01[K     |████▏                           | 276 kB 7.0 MB/s eta 0:00:01[K     |████▍                           | 286 kB 7.0 MB/s eta 0:00:01[K     |████▌                           | 296 kB 7.0 MB/s eta 0:00:01[K     |████▋                           | 307 kB 7.0 MB/s eta 0:00:01[K     |████▉                           | 317 kB 7.0 MB/s eta 0:00:01[K     |█████                           | 327 kB 7.0 MB/s eta 0:00:01[K     |█████▏                          | 337 kB 7.0 MB/s eta 0:00:01[K     |█████▎                          | 348 kB 7.0 MB/s eta 0:00:01[K     |█████▍                          | 358 kB 7.0 MB/s eta 0:00:01[K     |█████▋                          | 368 kB 7.0 MB/s eta 0:00:01[K     |█████▊                          | 378 kB 7.0 MB/s eta 0:00:01[K     |██████                          | 389 kB 7.0 MB/s eta 0:00:01[K     |██████                          | 399 kB 7.0 MB/s eta 0:00:01[K     |██████▏                         | 409 kB 7.0 MB/s eta 0:00:01[K     |██████▍                         | 419 kB 7.0 MB/s eta 0:00:01[K     |██████▌                         | 430 kB 7.0 MB/s eta 0:00:01[K     |██████▊                         | 440 kB 7.0 MB/s eta 0:00:01[K     |██████▉                         | 450 kB 7.0 MB/s eta 0:00:01[K     |███████                         | 460 kB 7.0 MB/s eta 0:00:01[K     |███████▏                        | 471 kB 7.0 MB/s eta 0:00:01[K     |███████▎                        | 481 kB 7.0 MB/s eta 0:00:01[K     |███████▌                        | 491 kB 7.0 MB/s eta 0:00:01[K     |███████▋                        | 501 kB 7.0 MB/s eta 0:00:01[K     |███████▊                        | 512 kB 7.0 MB/s eta 0:00:01[K     |████████                        | 522 kB 7.0 MB/s eta 0:00:01[K     |████████                        | 532 kB 7.0 MB/s eta 0:00:01[K     |████████▎                       | 542 kB 7.0 MB/s eta 0:00:01[K     |████████▍                       | 552 kB 7.0 MB/s eta 0:00:01[K     |████████▌                       | 563 kB 7.0 MB/s eta 0:00:01[K     |████████▊                       | 573 kB 7.0 MB/s eta 0:00:01[K     |████████▉                       | 583 kB 7.0 MB/s eta 0:00:01[K     |█████████                       | 593 kB 7.0 MB/s eta 0:00:01[K     |█████████▏                      | 604 kB 7.0 MB/s eta 0:00:01[K     |█████████▎                      | 614 kB 7.0 MB/s eta 0:00:01[K     |█████████▌                      | 624 kB 7.0 MB/s eta 0:00:01[K     |█████████▋                      | 634 kB 7.0 MB/s eta 0:00:01[K     |█████████▉                      | 645 kB 7.0 MB/s eta 0:00:01[K     |██████████                      | 655 kB 7.0 MB/s eta 0:00:01[K     |██████████                      | 665 kB 7.0 MB/s eta 0:00:01[K     |██████████▎                     | 675 kB 7.0 MB/s eta 0:00:01[K     |██████████▍                     | 686 kB 7.0 MB/s eta 0:00:01[K     |██████████▋                     | 696 kB 7.0 MB/s eta 0:00:01[K     |██████████▊                     | 706 kB 7.0 MB/s eta 0:00:01[K     |██████████▉                     | 716 kB 7.0 MB/s eta 0:00:01[K     |███████████                     | 727 kB 7.0 MB/s eta 0:00:01[K     |███████████▏                    | 737 kB 7.0 MB/s eta 0:00:01[K     |███████████▍                    | 747 kB 7.0 MB/s eta 0:00:01[K     |███████████▌                    | 757 kB 7.0 MB/s eta 0:00:01[K     |███████████▋                    | 768 kB 7.0 MB/s eta 0:00:01[K     |███████████▉                    | 778 kB 7.0 MB/s eta 0:00:01[K     |████████████                    | 788 kB 7.0 MB/s eta 0:00:01[K     |████████████▏                   | 798 kB 7.0 MB/s eta 0:00:01[K     |████████████▎                   | 808 kB 7.0 MB/s eta 0:00:01[K     |████████████▍                   | 819 kB 7.0 MB/s eta 0:00:01[K     |████████████▋                   | 829 kB 7.0 MB/s eta 0:00:01[K     |████████████▊                   | 839 kB 7.0 MB/s eta 0:00:01[K     |█████████████                   | 849 kB 7.0 MB/s eta 0:00:01[K     |█████████████                   | 860 kB 7.0 MB/s eta 0:00:01[K     |█████████████▏                  | 870 kB 7.0 MB/s eta 0:00:01[K     |█████████████▍                  | 880 kB 7.0 MB/s eta 0:00:01[K     |█████████████▌                  | 890 kB 7.0 MB/s eta 0:00:01[K     |█████████████▊                  | 901 kB 7.0 MB/s eta 0:00:01[K     |█████████████▉                  | 911 kB 7.0 MB/s eta 0:00:01[K     |██████████████                  | 921 kB 7.0 MB/s eta 0:00:01[K     |██████████████▏                 | 931 kB 7.0 MB/s eta 0:00:01[K     |██████████████▎                 | 942 kB 7.0 MB/s eta 0:00:01[K     |██████████████▌                 | 952 kB 7.0 MB/s eta 0:00:01[K     |██████████████▋                 | 962 kB 7.0 MB/s eta 0:00:01[K     |██████████████▊                 | 972 kB 7.0 MB/s eta 0:00:01[K     |███████████████                 | 983 kB 7.0 MB/s eta 0:00:01[K     |███████████████                 | 993 kB 7.0 MB/s eta 0:00:01[K     |███████████████▎                | 1.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████▍                | 1.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████▌                | 1.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████▊                | 1.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████▉                | 1.0 MB 7.0 MB/s eta 0:00:01[K     |████████████████                | 1.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████▏               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████▎               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████▌               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████▋               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████▉               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |█████████████████               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |█████████████████               | 1.1 MB 7.0 MB/s eta 0:00:01[K     |█████████████████▎              | 1.1 MB 7.0 MB/s eta 0:00:01[K     |█████████████████▍              | 1.1 MB 7.0 MB/s eta 0:00:01[K     |█████████████████▋              | 1.2 MB 7.0 MB/s eta 0:00:01[K     |█████████████████▊              | 1.2 MB 7.0 MB/s eta 0:00:01[K     |█████████████████▉              | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████              | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████▏             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████▍             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████▌             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████▋             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |██████████████████▉             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |███████████████████             | 1.2 MB 7.0 MB/s eta 0:00:01[K     |███████████████████▏            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |███████████████████▎            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |███████████████████▍            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |███████████████████▋            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |███████████████████▊            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |████████████████████            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |████████████████████            | 1.3 MB 7.0 MB/s eta 0:00:01[K     |████████████████████▏           | 1.3 MB 7.0 MB/s eta 0:00:01[K     |████████████████████▍           | 1.3 MB 7.0 MB/s eta 0:00:01[K     |████████████████████▌           | 1.4 MB 7.0 MB/s eta 0:00:01[K     |████████████████████▊           | 1.4 MB 7.0 MB/s eta 0:00:01[K     |████████████████████▉           | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████           | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████▏          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████▎          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████▌          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████▋          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████▊          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████          | 1.4 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████          | 1.5 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████▎         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████▍         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████▌         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████▊         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████▉         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████         | 1.5 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████▏        | 1.5 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████▎        | 1.5 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████▌        | 1.5 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████▋        | 1.6 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████▉        | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████        | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████        | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████▎       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████▍       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████▋       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████▊       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████▉       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████       | 1.6 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████▏      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████▍      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████▌      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████▋      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████▉      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████      | 1.7 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████▏     | 1.7 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████▎     | 1.7 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████▍     | 1.7 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████▋     | 1.8 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████▊     | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████     | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████     | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████▏    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████▍    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████▌    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████▊    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████▉    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████    | 1.8 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████▏   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████▎   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████▌   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████▋   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████▊   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████   | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████▎  | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████▍  | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████▌  | 1.9 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████▊  | 2.0 MB 7.0 MB/s eta 0:00:01[K     |█████████████████████████████▉  | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████  | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████▏ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████▎ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████▌ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████▋ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |██████████████████████████████▉ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████ | 2.0 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████▎| 2.1 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████▍| 2.1 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████▋| 2.1 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████▊| 2.1 MB 7.0 MB/s eta 0:00:01[K     |███████████████████████████████▉| 2.1 MB 7.0 MB/s eta 0:00:01[K     |████████████████████████████████| 2.1 MB 7.0 MB/s 
[?25hInstalling collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.0.2
    Not uninstalling pip at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'pip'. No files were found to uninstall.
Successfully installed pip-23.3.1
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q17_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q17_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q17_t0.sh: line 5: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q17_t0.sh: line 8: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q18_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q19_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
Calculating upgrade... 0%Calculating upgrade... 50%Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q1_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q1_t0.sh: line 2: syntax error near unexpected token `('
/platform/source/s01/v4_engoff/m11122200/m11122200_q1_t0.sh: line 2: `with open('install_dependencies.sh', 'w') as file:'
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mAll dependencies have been installed.
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q21_t0.sh
0% [Working]            Hit:1 http://ports.ubuntu.com/ubuntu-ports focal InRelease
0% [Working]0% [Waiting for headers]                        Hit:2 http://ports.ubuntu.com/ubuntu-ports focal-updates InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:3 http://ports.ubuntu.com/ubuntu-ports focal-backports InRelease
                        0% [Working]0% [Waiting for headers]                        Hit:4 http://ports.ubuntu.com/ubuntu-ports focal-security InRelease
                        0% [Working]20% [Working]             Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
Calculating upgrade... 0%Calculating upgrade... 50%Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3 is already the newest version (3.8.2-0ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Reading package lists... 0%Reading package lists... 0%Reading package lists... 0%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 6%Reading package lists... 60%Reading package lists... 60%Reading package lists... 61%Reading package lists... 61%Reading package lists... 74%Reading package lists... 74%Reading package lists... 75%Reading package lists... 75%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 81%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 82%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 94%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... 99%Reading package lists... Done
Building dependency tree... 0%Building dependency tree... 0%Building dependency tree... 50%Building dependency tree... 50%Building dependency tree       
Reading state information... 0%Reading state information... 0%Reading state information... Done
python3-pip is already the newest version (20.0.2-5ubuntu1.10).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q22_t0.sh
Running /platform/source/s01/v4_engoff/m11122200/m11122200_q2_t0.sh
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q3_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q3_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pip in /usr/local/lib/python3.8/dist-packages (23.3.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q4_t0.sh
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q5_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q5_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q5_t0.sh: line 5: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: redis in /usr/local/lib/python3.8/dist-packages (3.4.1)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q6_t0.sh
/platform/source/s01/v4_engoff/m11122200/m11122200_q6_t0.sh: line 4: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q6_t0.sh: line 7: sudo: command not found
/platform/source/s01/v4_engoff/m11122200/m11122200_q6_t0.sh: line 10: sudo: command not found
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q7_t0.sh
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRequirement already satisfied: direct_redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct_redis) (3.4.1)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q8_t0.sh
Requirement already satisfied: pymysql in /usr/local/lib/python3.8/dist-packages (1.1.0)
Requirement already satisfied: pandas in /usr/local/lib/python3.8/dist-packages (2.0.3)
Requirement already satisfied: pymongo in /usr/local/lib/python3.8/dist-packages (4.6.0)
Requirement already satisfied: direct-redis in /usr/local/lib/python3.8/dist-packages (0.3.1)
Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.8/dist-packages (from pandas) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3.post1)
Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.8/dist-packages (from pandas) (2023.3)
Requirement already satisfied: numpy>=1.20.3 in /usr/local/lib/python3.8/dist-packages (from pandas) (1.24.4)
Requirement already satisfied: dnspython<3.0.0,>=1.16.0 in /usr/local/lib/python3.8/dist-packages (from pymongo) (2.4.2)
Requirement already satisfied: redis==3.4.1 in /usr/local/lib/python3.8/dist-packages (from direct-redis) (3.4.1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.8.2->pandas) (1.16.0)
[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv[0m[33m
[0mRunning /platform/source/s01/v4_engoff/m11122200/m11122200_q9_t0.sh
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q10_t0.csv /platform/expected/s01/q10.csv
/platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.py:24: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  mysql_result = pd.read_sql_query(mysql_query, mysql_conn)
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q10_t0.py", line 43, in <module>
    customers_df = pd.read_json(customer_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q11_t0.csv /platform/expected/s01/q11.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.py", line 46, in <module>
    main()
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q11_t0.py", line 26, in main
    supplier_df = pd.read_json(redis_db.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q12_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q12_t0.csv /platform/expected/s01/q12.csv
>>> Expected ((2, 3))
   HIGH_LINE_COUNT  LOW_LINE_COUNT L_SHIPMODE
0              647             945       MAIL
1              620             943       SHIP

>>> Actual ((4, 3))
   LATE_COUNT      PRIORITY SHIPMODE
0         945  NON-CRITICAL     SHIP
1         946  NON-CRITICAL     MAIL
2         651      CRITICAL     MAIL
3         621      CRITICAL     SHIP

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q13_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q13_t0.csv /platform/expected/s01/q13.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q13_t0.py", line 50, in <module>
    main()
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q13_t0.py", line 43, in main
    redis_data = query_redis()
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q13_t0.py", line 38, in query_redis
    redis_df = pd.read_json(customer_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q14_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q14_t0.csv /platform/expected/s01/q14.csv
>>> Expected ((1, 1))
   PROMO_REVENUE
0      16.283856

>>> Actual ((1, 1))
   Promotion Revenue Percentage
0                             0

Head row similarity: 0.2708333333333333
Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q15_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q15_t0.csv /platform/expected/s01/q15.csv
/platform/source/s01/v4_engoff/m11122200/m11122200_q15_t0.py:32: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  lineitem_df = pd.read_sql(lineitem_query, mysql_conn)
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q15_t0.py", line 36, in <module>
    supplier_df = pd.read_json(supplier_encoded, orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q16_t0.csv /platform/expected/s01/q16.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.py", line 62, in <module>
    main()
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.py", line 49, in main
    partsupp_df = fetch_redis_data(redis_db)
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q16_t0.py", line 30, in fetch_redis_data
    supplier_data = rd.get('supplier')
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 1579, in get
    return self.execute_command('GET', name)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 878, in execute_command
    return self.parse_response(conn, command_name, **options)
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 892, in parse_response
    response = connection.read_response()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 734, in read_response
    response = self._parser.read_response()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 358, in read_response
    response = self.encoder.decode(response)
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 129, in decode
    value = value.decode(self.encoding, self.encoding_errors)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q17_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q17_t0.csv /platform/expected/s01/q17.csv
>>> Expected ((1, 1))
     AVG_YEARLY
0  31865.742857

>>> Actual ((1, 1))
   average_yearly_loss
0             36006.55

Head row similarity: 0.2894736842105263
Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q18_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q18_t0.csv /platform/expected/s01/q18.csv
/platform/source/s01/v4_engoff/m11122200/m11122200_q18_t0.py:24: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  mysql_data = pd.read_sql(query, mysql_conn)
>>> Expected ((5, 6))
   C_CUSTKEY              C_NAME O_ORDERDATE  O_ORDERKEY  O_TOTALPRICE  SUM(L_QUANTITY)
0       1639  Customer#000001639  1994-04-12      502886     456423.88              312
1       6655  Customer#000006655  1995-10-21       29158     452805.02              305
2      14110  Customer#000014110  1995-09-24      565574     425099.85              301
3       1775  Customer#000001775  1997-04-09        6882     408368.10              303
4      11459  Customer#000011459  1993-05-19      551136     386812.74              308

>>> Actual ((5, 6))
               C_NAME  O_CUSTKEY O_ORDERDATE  O_ORDERKEY  O_TOTALPRICE  total_quantity
0  Customer#000001775       1775  1997-04-09        6882     408368.10           303.0
1  Customer#000014110      14110  1995-09-24      565574     425099.85           301.0
2  Customer#000006655       6655  1995-10-21       29158     452805.02           305.0
3  Customer#000001639       1639  1994-04-12      502886     456423.88           312.0
4  Customer#000011459      11459  1993-05-19      551136     386812.74           308.0

Head row similarity: 0.3988549618320611
Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q19_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q19_t0.csv /platform/expected/s01/q19.csv
>>> Expected ((1, 1))
      REVENUE
0  168597.286

>>> Actual ((10, 2))
   PARTKEY     REVENUE
0     3762   1549.1568
1    19146  30036.9480
2    17233  16735.8465
3    19450  15283.0620
4    18282  22085.1520
5       55  15748.7745
6     3987  15657.3144
7    15576   4385.2158
8    12405  28772.0160
9    13106  18343.8000

Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q1_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q1_t0.csv /platform/expected/s01/q1.csv
Score.fail_missing_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q20_t0.csv /platform/expected/s01/q20.csv
/platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.py:28: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df_lineitem = pd.read_sql(query_lineitem, mysql_conn)
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q20_t0.py", line 35, in <module>
    df_supplier = pd.read_msgpack(supplier_docs)
AttributeError: module 'pandas' has no attribute 'read_msgpack'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q21_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q21_t0.csv /platform/expected/s01/q21.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q21_t0.py", line 33, in <module>
    supplier = pd.read_json(supplier_data, orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q22_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q22_t0.csv /platform/expected/s01/q22.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q22_t0.py", line 33, in <module>
    customer_df = pd.read_json(customer_data, orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q2_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q2_t0.csv /platform/expected/s01/q2.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q2_t0.py", line 31, in <module>
    df_european_supplier_parts = pd.merge(df_partsupp, df_european_suppliers, left_on='S_SUPPKEY', right_on='S_SUPPKEY')
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 148, in merge
    op = _MergeOperation(
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 737, in __init__
    ) = self._get_merge_keys()
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/reshape/merge.py", line 1221, in _get_merge_keys
    left_keys.append(left._get_label_or_level_values(lk))
  File "/usr/local/lib/python3.8/dist-packages/pandas/core/generic.py", line 1778, in _get_label_or_level_values
    raise KeyError(key)
KeyError: 'S_SUPPKEY'
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q3_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q3_t0.csv /platform/expected/s01/q3.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q3_t0.py", line 33, in <module>
    cursor.execute(query)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/cursors.py", line 153, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 558, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 822, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 1200, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 772, in _read_packet
    packet.raise_for_error()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1055, "Expression #2 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'tpch.o.O_SHIPPRIORITY' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by")
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q4_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q4_t0.csv /platform/expected/s01/q4.csv
>>> Expected ((5, 2))
   ORDER_COUNT  O_ORDERPRIORITY
0          999         1-URGENT
1          997           2-HIGH
2         1031         3-MEDIUM
3          989  4-NOT SPECIFIED
4         1077            5-LOW

>>> Actual ((5, 2))
   O_ORDERPRIORITY  order_count
0         1-URGENT          999
1           2-HIGH          997
2         3-MEDIUM         1031
3  4-NOT SPECIFIED          989
4            5-LOW         1077

Head row similarity: 0.7539267015706806
Score.success_approx
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q5_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q5_t0.csv /platform/expected/s01/q5.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q5_t0.py", line 34, in <module>
    supplier = pd.read_json(supplier_data)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q6_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q6_t0.csv /platform/expected/s01/q6.csv
>>> Expected ((1, 1))
        REVENUE
0  7.115407e+06

>>> Actual ((1, 1))
   potential_revenue_increase
0                1.180342e+07

Head row similarity: 0.35555555555555557
Score.fail_wrong_output
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q7_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q7_t0.csv /platform/expected/s01/q7.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q7_t0.py", line 26, in <module>
    supplier_data = pd.read_json(redis_conn.get('supplier'), orient='records')
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q8_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q8_t0.csv /platform/expected/s01/q8.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q8_t0.py", line 66, in <module>
    main()
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q8_t0.py", line 50, in main
    supplier_data = get_redis_data(redis_conn_info)
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q8_t0.py", line 40, in get_redis_data
    supplier = pd.read_json(direct_redis_conn.get('supplier'))
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 760, in read_json
    json_reader = JsonReader(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 861, in __init__
    data = self._get_data_from_filepath(filepath_or_buffer)
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/json/_json.py", line 901, in _get_data_from_filepath
    self.handles = get_handle(
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 707, in get_handle
    if _is_binary_mode(path_or_buf, mode) and "b" not in mode:
  File "/usr/local/lib/python3.8/dist-packages/pandas/io/common.py", line 1167, in _is_binary_mode
    return isinstance(handle, _get_binary_io_classes()) or "b" in getattr(
TypeError: argument of type 'method' is not iterable
Score.fail_execute
python /dbknitter/dbknitter/grader.py /platform/source/s01/v4_engoff/m11122200/m11122200_q9_t0.py /platform/output/s01/v4_engoff/m11122200/m11122200_q9_t0.csv /platform/expected/s01/q9.csv
Traceback (most recent call last):
  File "/platform/source/s01/v4_engoff/m11122200/m11122200_q9_t0.py", line 33, in <module>
    mysql_cursor.execute(mysql_query)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/cursors.py", line 153, in execute
    result = self._query(query)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 558, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 822, in _read_query_result
    result.read()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 1200, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/connections.py", line 772, in _read_packet
    packet.raise_for_error()
  File "/usr/local/lib/python3.8/dist-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python3.8/dist-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'tpch.supplier' doesn't exist")
Score.fail_execute
